_includes/selector.html

```diff
@@ -483,6 +495,26 @@
 
                 return lines;
             },
+            getPipCmdHtml() {
+                var pip_install = "pip install ";
+                var index_url = " --extra_index_url=https://pypi.nvidia.com";
+                var pkgs = String(this.active_packages).replaceAll(",", "-cu11 ").toLowerCase() + "-cu11";
+                var isArm = this.active_arch === "arm";
+                var cupy_inst = "";
+
+                if (pkgs.includes("cucim")) {
+                    pkgs = pkgs.replaceAll("cucim-cu11", "cucim");
+                }
+                if (pkgs === "cucim") {
+                    index_url = "";
+                }
+
+                if (isArm) {
+                    cupy_inst = "\npip install cupy-cuda11x -f https://pip.cupy.dev/aarch64";
+                }
```

Comment:
Can we make a note that this should go away in 23.06? Don't want to lose track of it.

Reply:
Will do